build.gradle

```diff
@@ -10,13 +10,11 @@ plugins {
 apply plugin: 'project-report'
 
 description = 'JGiven - BDD in plain Java'
-def jacocoEnabled = !JavaVersion.current().isJava11Compatible();
-
-
+def jacocoEnabled = !JavaVersion.current().isJava11Compatible()
 
 wrapper {
```

Comment:
`wrapper` section should not be needed and is not part of at least my repository anymore, see e.g. https://github.com/aaschmid/gradle-cpd-plugin/blob/master/build.gradle

Reply:
giving it some thought, I guess you're right. After all this section only replicates the information that we have already checked in in the `gradle-wrapper.properties` file.